.circleci/config.yml

```diff
@@ -0,0 +1,382 @@
+version: 2.0
+
+#### YAML ANCHORS INSTEAD OF COMMANDS # while version 2.1 already supports commands, the local runner doesn't. When it does, we can upgrade.
+
+git-submodules: &git-submodules
+  run:
+    name: "Pull Submodules"
+    command: |
+      git submodule init
+      git submodule update
+
+store-junit-results: &store-junit-test-results
+  store_test_results:
+    path: tmp/junit
+
+store-test-artifacts: &store-test-artifacts
+  store_artifacts:
+    path: tmp/junit
+    destination: junit
+
+store-log-artifacts: &store-log-artifacts
+  store_artifacts:
+    path: log/test.log
+    destination: test.log
+
+restore-bundler-cache: &restore-bundler-cache
+  restore_cache:
+    keys:
+      - v1-bundler-gems-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+restore-npm-cache: &restore-npm-cache
+  restore_cache:
+    keys:
+      - v1-npm-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}
+#restore-gateway-cache: &restore-gateway-cache
+#  restore_cache:
+#    keys:
+#      - v1-gateway-{{ arch }}-{{ .Branch }}-{{ .Revision }}
+
+
+
+attach-to-workspace: &attach-to-workspace
+  attach_workspace:
+    at: .
+
+system-builder: &system-builder
+  image: quay.io/gsaslis/3scale-system-builder:latest
```

Comment:
Shouldn't this be pushed to `quay.io/3scale` instead? @gsaslis @mikz

Reply:
Yes, it will be at some point.